packages/cactus-test-tooling/src/main/typescript/web-socket-test-client/web-socket-key.ts

```diff
@@ -0,0 +1,87 @@
+import { KEYUTIL } from "jsrsasign";
+import fs from "fs";
+const jsrsasign = require("jsrsasign");
+
+import { LoggerProvider } from "@hyperledger/cactus-common";
+
+export enum ECCurveType {
```

Comment:
@brioux Can these be pulled from a library instead of redefining the constants?

Reply:
these constant are also defined in the src/main/typescript/identity/internal/crypto-util. 
The enum is used in [ws-wallet](https://github.com/brioux/ws-wallet). Will identify a library to consolidate these

Reply:
@petermetz  i have fixed the unused dependencies in cactus-smd-socketio-server